### PR TITLE
notifications: improve acquisition notification template

### PR DESCRIPTION
**Pull request by zannkukai**
* Adds some metadata into the available keys dictionary to create the
  acquisition order notification.
* Updates acquisition order notification templates with more metadata.
* Cleans some code statements using "sourcery" plugin.

Co-Authored-by: Renaud Michotte <renaud.michotte@gmail.com>

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Cypress tests successful?

### CODE REVIEW

File: rero_ils/modules/acq_orders/templates/rero_ils/vendor_order_mail/eng.tpl.txt [hunk 1]
```diff
@@ -9,14 +9,24 @@ We would like to order the following list of books:
...
 ------------------------------------------------------------
 {%- for line in order.order_lines %}
-Title : {{ line.document.title }}
-{%- if line.document.identifiers %}
-ISBN : {{ line.document.identifiers | join(', ') }}
-{%- endif %}
+Document :
+  {{ line.document.title_text }}
+  {%- if line.document.provision_activity %}
+  Provision activity : {{ line.document.provision_activity }}
```
Comment (pronguen) [inline rero_ils/modules/acq_orders/templates/rero_ils/vendor_order_mail/eng.tpl.txt:15]: In 99% of the cases, the provision activity is a publication activity, so it's clearer to put here Publication statement.

```suggestion
  Publication statement : {{ line.document.provision_activity }}
```

File: rero_ils/modules/acq_orders/templates/rero_ils/vendor_order_mail/eng.tpl.txt [hunk 2]
```diff
@@ -9,14 +9,24 @@ We would like to order the following list of books:
...
+  {%- endif %}
+  {%- if line.document.serie_statement %}
+  Collection : {{ line.document.serie_statement }}
```
Comment (pronguen) [inline rero_ils/modules/acq_orders/templates/rero_ils/vendor_order_mail/eng.tpl.txt:18]:
```suggestion
  Series : {{ line.document.serie_statement }}
```

File: rero_ils/modules/acq_orders/templates/rero_ils/vendor_order_mail/fre.tpl.txt [hunk 3]
```diff
@@ -9,14 +9,24 @@ Nous souhaitons vous commander la liste d'ouvrages suivants:
...
 ------------------------------------------------------------
 {%- for line in order.order_lines %}
-Titre : {{ line.document.title }}
-{%- if line.document.identifiers %}
-ISBN : {{ line.document.identifiers | join(', ') }}
-{%- endif %}
+Document :
+  {{ line.document.title_text }}
+  {%- if line.document.provision_activity %}
+  Mention : {{ line.document.provision_activity }}
```
Comment (pronguen) [inline rero_ils/modules/acq_orders/templates/rero_ils/vendor_order_mail/fre.tpl.txt:15]:
```suggestion
  Mention de publication : {{ line.document.provision_activity }}
```

File: rero_ils/modules/acq_orders/templates/rero_ils/vendor_order_mail/ger.tpl.txt [hunk 4]
```diff
@@ -9,14 +9,24 @@ Wir möchten die folgende Liste von Büchern bestellen:
...
 ------------------------------------------------------------
 {%- for line in order.order_lines %}
-Titel : {{ line.document.title }}
-{%- if line.document.identifiers %}
-ISBN : {{ line.document.identifiers | join(', ') }}
-{%- endif %}
+Dokument :
+  {{ line.document.title_text }}
+  {%- if line.document.provision_activity %}
+  Veröffentlichung, Herstellung, Vertrieb, Entstehung : {{ line.document.provision_activity }}
```
Comment (pronguen) [inline rero_ils/modules/acq_orders/templates/rero_ils/vendor_order_mail/ger.tpl.txt:15]:
```suggestion
  Veröffentlichungsangabe : {{ line.document.provision_activity }}
```

File: rero_ils/modules/acq_orders/templates/rero_ils/vendor_order_mail/ita.tpl.txt [hunk 5]
```diff
@@ -9,14 +9,24 @@ Vorremmo ordinare la seguente lista di libri:
...
 ------------------------------------------------------------
 {%- for line in order.order_lines %}
-Titolo : {{ line.document.title }}
-{%- if line.document.identifiers %}
-ISBN : {{ line.document.identifiers | join(', ') }}
-{%- endif %}
+Documento :
+  {{ line.document.title_text }}
+  {%- if line.document.provision_activity %}
+  Pubblicazione, manifattura, distribuzione, produzione : {{ line.document.provision_activity }}
```
Comment (pronguen) [inline rero_ils/modules/acq_orders/templates/rero_ils/vendor_order_mail/ita.tpl.txt:15]: Official translation of RDA (https://original.rdatoolkit.org/)

```suggestion
  Formulazione di pubblicazione : {{ line.document.provision_activity }}
```

File: rero_ils/modules/acq_orders/templates/rero_ils/vendor_order_mail/ger.tpl.txt [hunk 6]
```diff
@@ -9,14 +9,24 @@ Wir möchten die folgende Liste von Büchern bestellen:
...
+  {%- endif %}
+  {%- if line.document.serie_statement %}
+  Sammlung : {{ line.document.serie_statement }}
```
Comment (pronguen) [inline rero_ils/modules/acq_orders/templates/rero_ils/vendor_order_mail/ger.tpl.txt:18]:
```suggestion
  Reihe : {{ line.document.serie_statement }}
```

File: rero_ils/modules/acq_orders/templates/rero_ils/vendor_order_mail/ger.tpl.txt [hunk 7]
```diff
@@ -9,14 +9,24 @@ Wir möchten die folgende Liste von Büchern bestellen:
...
+  {%- endif %}
+  {%- if line.document.identifiers %}
+  ISBN : {{ line.document.identifiers | join(', ') }}
+  {%- endif %}
 {%- if line.note %}
 Hinweis : {{ line.note }}
```
Comment (pronguen) [inline rero_ils/modules/acq_orders/templates/rero_ils/vendor_order_mail/ger.tpl.txt:24]:
```suggestion
Anmerkung : {{ line.note }}
```

File: rero_ils/modules/acq_orders/templates/rero_ils/vendor_order_mail/ger.tpl.txt [hunk 8]
```diff
@@ -9,14 +9,24 @@ Wir möchten die folgende Liste von Büchern bestellen:
...
 {%- endif %}
 Menge : {{ line.quantity }}
```
Comment (pronguen) [inline rero_ils/modules/acq_orders/templates/rero_ils/vendor_order_mail/ger.tpl.txt:26]:
```suggestion
Anzahl : {{ line.quantity }}
```

File: rero_ils/modules/acq_orders/templates/rero_ils/vendor_order_mail/ger.tpl.txt [hunk 9]
```diff
@@ -9,14 +9,24 @@ Wir möchten die folgende Liste von Büchern bestellen:
...
+lokales Konto : {{ line.account.name }} {% if line.account.number: -%}
```
Comment (pronguen) [inline rero_ils/modules/acq_orders/templates/rero_ils/vendor_order_mail/ger.tpl.txt:27]:
```suggestion
Lokales Konto : {{ line.account.name }} {% if line.account.number: -%}
```

File: rero_ils/modules/acq_orders/templates/rero_ils/vendor_order_mail/ita.tpl.txt [hunk 10]
```diff
@@ -9,14 +9,24 @@ Vorremmo ordinare la seguente lista di libri:
...
+  {%- endif %}
+  {%- if line.document.serie_statement %}
+  Collezione : {{ line.document.serie_statement }}
```
Comment (pronguen) [inline rero_ils/modules/acq_orders/templates/rero_ils/vendor_order_mail/ita.tpl.txt:18]:
```suggestion
  Serie : {{ line.document.serie_statement }}
```

**Review by pronguen — COMMENTED** [review]
@vgranata @rerowep Je pense que les traductions sont bonnes selon mes demandes de modifications. Je me suis basé sur RDA.

File: rero_ils/modules/acq_orders/templates/rero_ils/vendor_order_mail/ita.tpl.txt (hunk 5 above)
Comment (vgranata) [reply rero_ils/modules/acq_orders/templates/rero_ils/vendor_order_mail/ita.tpl.txt]: OK for me

File: rero_ils/modules/acq_orders/templates/rero_ils/vendor_order_mail/ita.tpl.txt (hunk 10 above)
Comment (vgranata) [reply rero_ils/modules/acq_orders/templates/rero_ils/vendor_order_mail/ita.tpl.txt]: OK for me

File: rero_ils/modules/acq_orders/templates/rero_ils/vendor_order_mail/eng.tpl.txt (hunk 1 above)
Comment (vgranata) [reply rero_ils/modules/acq_orders/templates/rero_ils/vendor_order_mail/eng.tpl.txt]: OK for me

File: rero_ils/modules/acq_orders/templates/rero_ils/vendor_order_mail/eng.tpl.txt (hunk 2 above)
Comment (vgranata) [reply rero_ils/modules/acq_orders/templates/rero_ils/vendor_order_mail/eng.tpl.txt]: OK for me

**Review by BadrAly — APPROVED** [review]

**Pull request merged**